src/schema/objects/metadata.yaml

```diff
@@ -356,6 +374,47 @@ BrainLocation:
     or layer to refer to layer-specific gene expression,
     which can also tie up with laminar fMRI.
   type: string
+CalibrationList:
+  name: CalibrationList
+  display_name: Calibration List
+  description: |
+    List of lists including information for each calibration.
+    This list includes the calibration type, recorded eye,
+    maximal calibration error, average calibration error,
+    and time relative to the first event of the event file.
+  type: object
+  items:
+    type: object
+CalibrationPosition:
+  name: CalibrationPosition
+  display_name: Calibration Position
+  description: |
+    Provide a list of X/Y coordinates in the `CalibrationUnit`.
```

Comment:
Since this is a list of two-coordinate lists, you could restrict the internal lists to two elements. Also, at the moment, it looks like this is a list of numbers, rather than a list of lists of numbers.